workers/unity/Packages/io.improbable.gdk.core/Systems/SpatialOSReceiveSystem.cs

```diff
@@ -31,17 +31,17 @@ protected override void OnUpdate()
         {
             try
             {
-                worker.Tick();
+                workerSystem.Tick();
 
-                var diff = worker.Diff;
-                worker.View.ApplyDiff(diff);
+                var diff = workerSystem.Diff;
+                workerSystem.ApplyDiff(diff);
```

Comment:
This feels a little weird. Shouldn't `Tick` just do `ApplyDiff` internally?

Reply:
I explicitly didn't do this to keep all places where the diff is applied in 1 place.
This could later change to some pubsub system where the workerSystem publishes the new diff and others system subscribe and apply it

Reply:
If we do https://github.com/spatialos/gdk-for-unity/pull/1364#discussion_r464977855 instead it may fix itself?